Spotify/form.json

```diff
@@ -67,7 +67,7 @@
                     "type": "Button",
                     "name": "Search",
                     "caption": "Start Search",
-                    "onClick": "SPO_Search($id, $SearchQuery, $SearchAlbums, $SearchArtists, $SearchPlaylists, $SearchTracks);"
+                    "onClick": "SPO_Search_Configform($id, $SearchQuery, $SearchAlbums, $SearchArtists, $SearchPlaylists, $SearchTracks);"
```

Comment:
Ich bin kein Freund des Unterstrichs im Funktionsnamen. Üblicherweise benennen wir die Funktionen in Camelcase, also bitte stattdessen SPO_SearchConfigForm